### DISCUSSION TITLE
Script integrati in google home

### DISCUSSION THREAD

**Discussion by tabs78** [indomus/forum#2887] [GENERALE - Discussioni generiche]
Ciao. 
ho una doppia richiesta. 
ho integrato ha in google home col metodo gratuito.
Ho messo nei domini integrati gli script ma alcuni non li vedo.
C'è un motivo?

collegato a questo come si fa a integrare entità soecifichey?
Ho
Messo
La stringa entities_config ma non si può semplicemente mettere la lista cosi. Mi da errore, vuole l'argomento per ognuno

**Comment by snakuzzo** [comment]
Che vuol dire "alcuni non li vedo" ?
Tutti gli script non li visualizzerai mai come dispositivi, ma li troverai quando andrai a creare delle routine.
A tal proposito leggi [QUI](https://indomus.it/progetti/eseguire-sequenze-logiche-sulla-domotica-home-assistant-tramite-comandi-vocali-google-nest/) ... è tutto spiegato

Non so se hai sbagliato a scrivere o cosa, ma non si capisce la tua seconda richiesta. A parte che hai proprio sbagliato a scrivere, nella nostra guida [QUI](https://indomus.it/guide/integrare-gratuitamente-google-home-assistant-con-home-assistant-via-gcp/#configurazionehomeassistant) è spiegato tutto
